web3/_utils/module_testing/go_ethereum_txpool_module.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+
+from web3 import Web3
+
+
+class GoEthereumTxPoolModuleTest:
```

Comment:
I switched the name but I used `GoEthereumAsyncTxPoolModuleTest` because it matched the naming used in the rest of `test_gothereum_http.py` I hope that is works.